.travis.yml

```diff
@@ -2,9 +2,8 @@ language: php
 
 php:
   - hhvm
+  - 7.0
```

Comment:
Not supporting EOL PHP versions just be cause they are EOL doesn't make any sense. Code is working on them and doesn't cost anything extra to support them.